code/game/objects/effects/contraband.dm

```diff
@@ -595,4 +600,15 @@
 	desc = "This informational poster teaches the viewer what carbon dioxide is."
 	icon_state = "poster35_legit"
 
+/obj/structure/sign/poster/official/spiderlings
+	name = "Spiderlings"
+	desc = "This poster helps inform the crew of the dangers of spiderlings."
+	icon_state = "poster36_legit"
+
+/obj/structure/sign/poster/official/duelshotgun
+	name = "Cycler Shotgun Ad"
+	desc = "A advertisement poster of the latest breakthrough in shotgun technology, double barrable combat shotgun. \
+			The poster boasts about how easy it is to swap between the two cyclers."
```

Comment:
```suggestion
	desc = "A poster advertising an advanced dual magazine tubes shotgun, boasting about how easy it is to swap between the two tubes."
```
Wouldn't really call "breakthrough technology" something that can be dated back to the 20th century. Also some typos and semantics. :clown_face: